docs/getting_started.rst

```diff
@@ -363,259 +129,15 @@ that have their column ``delete`` set to ``1``::
             model = Book
 
 
-Signals
```

Comment:
Same q on this section, not sure where it went to. Or is it not relevant anymore?

Reply:
Also moved to ['advanced usage'](https://github.com/django-import-export/django-import-export/pull/1503/files#diff-151a99faebb7c20650eb815a2821b6613090470c2873994c64334fc9190ee1b0R441)